src/main/java/org/betonquest/betonquest/compatibility/holographicdisplays/lines/TextLine.java

```diff
@@ -0,0 +1,31 @@
+package org.betonquest.betonquest.compatibility.holographicdisplays.lines;
+
+import lombok.Getter;
+import me.filoghost.holographicdisplays.api.hologram.Hologram;
+
+/**
+ * Displays a simple text line with optional color codes
+ */
+@SuppressWarnings("PMD.CommentRequired")
```

Comment:
New Classes should not have any `@SuppressWarnings`, except false positives

And `CommentRequired` is not allowed at all for new Classes.